webui/Makefile

```diff
@@ -31,7 +34,7 @@ build-proto:
 
 .PHONY: build-api-ts-sdk
 build-api-ts-sdk: build-proto
-	$(MAKE) -C api-ts-sdk build
+	$(MAKE) -C api-ts-sdk get-deps build
```

Comment:
ah, I'm guessing is this why `tsc` was not available when building `api-ts-sdk`?

Reply:
exactly but also clean was missing so lack of get-deps wasn't showing